Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update \
   && echo '#!/bin/sh -e\nexec /usr/bin/node /usr/bin/uglifyjs "$@"' >/tmp/uglifyjs \
   && chmod a+x /tmp/uglifyjs && mv -t /usr/local/bin /tmp/uglifyjs
 
-RUN pip3 --no-cache-dir install pytest pytest-xdist pytest-instafail pytest-rerunfailures pytest-httpserver selenium PyYAML flake8
+RUN pip3 --no-cache-dir install pytest pytest-xdist pytest-instafail pytest-rerunfailures pytest-httpserver selenium PyYAML flake8 black distlib
```

Comment:
Might as well add mypy, pytest-cov Cython and Tempita to this, which we reinstall every time